@here/harp-materials/lib/MapMeshMaterials.ts

```diff
@@ -60,6 +60,16 @@ export interface ExtrusionFeatureParameters {
     zFightingWorkaround?: boolean;
 }
 
+/**
+ * Parameters used when constructing a [[MapMeshFlatStandardMaterial]].
+ */
+export interface ShadowParameters {
+    /**
+     * How strong the shadow is, 1 means complete black and 0 means no shadow shown.
+     */
+    shadowIntensity?: number;
```

Comment:
So `shadowIntensity` is actually a material property. Why is it then not defined in the fill technique, but in the light when looking into the theme?
If the reason is, that you want to avoid patching the theme file we should add it to all styles Berlin theme so that you don't have to patch the theme

Reply:
I don't see the need for such a level of customization, if the designers want to lighten the shadows, they would need to change it in 100 separate places, do we need this flexibility?

Reply:
you can still have a global variable in the style that is refered. So you still get the full flexibility and the ease of use.

Reply:
I'll answer on the previous comment, so will resolve this.